samples/msal-react-samples/gatsby-sample/README.md

```diff
@@ -0,0 +1,63 @@
+# MSAL.js for Gatsby Sample - Authorization Code Flow in Single-Page Applications
+
+## About this sample
+
```

Comment:
Nitpick: Might be good to have a short blurb about Gatsby or link to the Gatsby homepage (other than the link for the hello world git repo).

Reply:
It's at the bottom under "Learn more about the 3rd-party libraries used to create this sample" :)